### ISSUE TITLE
Add Electron support

### ISSUE THREAD

**Issue by ctmorrison** [barakwei/ParticlePapertrail#2]
Veryfing code using this library in Particle Build fails with this error:
```
papertrail/papertrail.cpp:11:26: error: 'WiFi' was not declared in this scope
     IPAddress remoteIP = WiFi.resolve(m_host)

I tried compiling the example and then I tried adding it to a small app I had created for a Photon.  I initially thought it might be an Electron incompatibility issue and then I got the same error when trying to compile it for a Photon.

**Comment by ctmorrison** [comment]
My mistake with the Photon.  I had forgotten to choose it as the target when compiling.  build was still pointing to the Electron.  Sorry.

**Comment by barakwei** [comment]
Thanks for reporting this. I don't have an Electron, so I didn't support it in the first version.
For some reason, the Electron doesn't have [Cellular](https://docs.particle.io/reference/firmware/electron/#cellular).resolve() method, so I'll have to find another way around this.

**Comment by barakwei** [comment]
Can you try using the code in the electron-support branch?

**Comment by barakwei** [comment]
This is fixed in 0.0.2
